python/benchmarks/single-node-io.py

```diff
@@ -361,7 +361,7 @@ def parse_directory(x):
         args.api = tuple(API.keys())
 
     # Create a temporary directory if user didn't specify a directory
-    temp_dir: Union[tempfile.TemporaryDirectory, contextlib.nullcontext]
+    temp_dir: Union[tempfile.TemporaryDirectory, ContextManager]
```

Comment:
mypy views `contextlib.nullcontext` as a function instead of a class and throws an error here - opted to use `ContextManager` instead to circumvent this, not sure if we can get more specific here

Reply:
`ContextManager` is fine, thanks